api/store/mongo/migrations.go

```diff
@@ -17,7 +17,7 @@ import (
 func ApplyMigrations(db *mongo.Database) error {
 	logrus.Info("Creating lock for the resource migrations")
 
-	lockClient := lock.NewClient(db.Collection("locks", options.Collection().SetWriteConcern(writeconcern.New(writeconcern.WMajority()))))
+	lockClient := lock.NewClient(db.Collection("locks", options.Collection().SetWriteConcern(writeconcern.New(writeconcern.WMajority())))) //nolint:staticcheck
```

Comment:
Why this is necessary?

Reply:
Added a nolint comment because github was failing the API validation with the  error: "SA1019: writeconcern.New is deprecated..."